app/views/tag_set_nominations/_tag_nominations_by_fandom.html.erb

```diff
@@ -9,20 +9,20 @@
             <%= nom.tagname %>  <%= nomination_status(nom) %>
           <% else %>
             <div title="<%= ts("Fandom %{index}") %>"><%= nom_form.text_field :tagname, autocomplete_options("fandom", :autocomplete_token_limit => 1, :class => "autocomplete") %></div>
-          <% end %>     
+          <% end %>
         </dd>
 
         <% # if fandoms are being nominated then any character/relationship noms go under fandom %>
         <% %w(character relationship).each do |tag_type| %>
           <% if @limit[tag_type] > 0 %>
             <%= nom_form.fields_for "#{tag_type}_nominations".to_sym do |nom_form2| %>
-              <dt><%= nom_form2.label :tagname, ts("#{tag_type.classify}") %>
+              <dt><%= nom_form2.label :tagname, tag_type_label_name(tag_type) %>
               <dd>
                 <% if nom_form2.object.approved  || nom_form2.object.rejected %>
                   <%= nom_form2.object.tagname %> <%= nomination_status(nom_form2.object) %>
                 <% else %>
-                  <div title="<%= ts("#{tag_type.classify}") %>"><%= nom_form2.text_field :tagname, 
-                            autocomplete_options("#{tag_type}_in_fandom", :autocomplete_token_limit => 1, 
+                  <div title="<%= tag_type_label_name(tag_type) %>"> <%= nom_form2.text_field :tagname,
```

Comment:
Should this be t, I suspect not.

Reply:
You're right, it should be translatable -- it's a title attribute that will be shown to users.
```suggestion
                  <div title="<%= ts("%{tag_type}", tag_type: tag_type_label_name(tag_type)) %>"> <%= nom_form2.text_field :tagname,
```